controller/Dockerfile

```diff
@@ -48,7 +48,7 @@ RUN if [ "$GOOS" = "linux" ] ; then \
 FROM golang AS eBPFRetinaStage
 WORKDIR /tmp
 RUN tdnf install -y unzip && \
-    curl -L -o eBPFRetina.zip https://www.nuget.org/api/v2/package/Microsoft.Wcn.Observability.eBPF.Retina.x64/0.1.0-prerelease.9 && \
+    curl -L -o eBPFRetina.zip https://www.nuget.org/api/v2/package/Microsoft.Wcn.Observability.eBPF.Retina.x64/0.1.0-prerelease.11 && \
```

Comment:
do we have any docs that we can attach for this zip?

Reply:
Not at this time - we can take that offline to see what we can do about docs.

Also note that the package does contain a header, which can help us understand what changed between the versions, which can be used to correlate back to the changes we have made in this PR.